azext_edge/tests/edge/checks/int/test_pre_post_int.py

```diff
@@ -7,39 +7,16 @@
 import pytest
 from knack.log import get_logger
 from azure.cli.core.azclierror import CLIInternalError
-from azext_edge.edge.providers.check.common import ResourceOutputDetailLevel
-from azext_edge.edge.providers.edge_api import (
-    AkriResourceKinds,
-    DataProcessorResourceKinds,
-    LnmResourceKinds,
-    MqResourceKinds,
-    OpcuaResourceKinds,
-)
-from ...helpers import run
+from ....helpers import run
 
 logger = get_logger(__name__)
 
 
-@pytest.mark.parametrize("detail_level", ResourceOutputDetailLevel.list())
```

Comment:
![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/73560279/50fdfe95-1c86-4e83-ae96-e5a38700ae8f)

the specific service checks should have detail level (assuming that detail level would have each `--as-object` result be different.